src/studio/src/designer/frontend/packages/schema-editor/src/components/SchemaInspector/ItemDataComponent.tsx

```diff
@@ -265,56 +265,58 @@ export function ItemDataComponent({
       );
     }
   };
-
+  const __ = (key: string) => getTranslation(key, language);
```

Comment:
Is the idea that this could be set globally at some point, to avoid passing the `language` prop down to all the components?
I like that it looks a lot cleaner, although it is less readable imo. Where does the `__` naming convention come from?

Reply:
Vi endret `__` til `t`, de doble underscorsa kommer fra PHP/gettext verden. gammel unix/linux greie. Enig i at den er confusing.